packages/flutter_tools/test/base/build_test.dart

```diff
@@ -96,14 +100,22 @@ void main() {
     setUp(() async {
       fs = MemoryFileSystem();
       fs.file(kSnapshotDart).createSync();
-      fs.file('.packages').writeAsStringSync('sky_engine:file:///flutter/bin/cache/pkg/sky_engine/lib/');
-
-      skyEnginePath = fs.path.fromUri(Uri.file('/flutter/bin/cache/pkg/sky_engine'));
-      fs.directory(fs.path.join(skyEnginePath, 'lib', 'ui')).createSync(recursive: true);
-      fs.directory(fs.path.join(skyEnginePath, 'sdk_ext')).createSync(recursive: true);
+      fs.file('.packages').writeAsStringSync(
```

Comment:
nit: Same comment as on `gradle_plugin_test.dart` on reverting auto formatting changes for this file. I can't actually spot any real logic changes here through the formatting diff.